src/nvim/fileio.c

```diff
@@ -3309,7 +3309,13 @@ static void vim_mktempdir(void)
     // "/tmp/" exists, now try to create "/tmp/nvim.<user>/".
     add_pathsep(tmp);
 
-    const char *appname = get_appname();
+    // Make a copy of the appname to avoid modifying it
+    char appname[40] = { 0 };
```

Comment:
where is 40 coming from?

Reply:
Similar to the above comment, I heavily referenced the way name worked when writing this 
[Seen here](https://github.com/neovim/neovim/blob/master/src/nvim/fileio.c#L3292)